lib/rules/no-extra-parens.js

```diff
@@ -431,6 +435,10 @@ module.exports = {
                     return;
                 }
 
+                if (node.body.type === "ConditionalExpression" && ENFORCE_FOR_ARROW_CONDITIONALS) {
+                    return;
+                }
```

Comment:
@not-an-aardvark I do not understand you a little. `enforceForArrowConditionals` default `false`? We change name option for consistently https://github.com/eslint/eslint/pull/8439#issuecomment-293021235. If `enforceForArrowConditionals` is true, we should not reported about extra parens in `var a = (b) => (1 ? 2 : 3)`, right?